bedrock/firefox/templates/firefox/switch.html

```diff
@@ -0,0 +1,93 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base-pebbles.html" %}
+
+{% block page_title %}{{ _('Switch from Chrome to Firefox in just a few minutes') }}{% endblock %}
+{% block page_desc %}{{ _('Switching to Firefox is fast, easy and risk-free, because Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.') }}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox-switch' %}
+{% endblock %}
+
+{% block body_id %}firefox-switch{% endblock %}
+
+{% block optimizely %}
+  {% if switch('firefox-switch-optimizely', ['en-US']) %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block content %}
+{% include 'firefox/includes/hub/sub-nav.html' %}
+<main>
+  <div class="content">
+    <header>
+      <h1>{{ _('Switch from Chrome to Firefox in just a few minutes') }}</h1>
+      <p class="tagline">{{ _('Switching to Firefox is fast, easy and risk-free. Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.') }}</p>
+    </header>
+
+    <ol class="prose">
+      <li>{{ _('Select what to take from Chrome.') }}</li>
+      <li>{{ _('Let Firefox do the rest.') }}</li>
+      <li>{{ _('Enjoy the web faster, all set up for you.') }}</li>
+    </ol>
+    {{ download_firefox(alt_copy=_('Download and switch'), download_location='primary cta') }}
+
+    <footer>
+      <h2>{{ _('Note:') }}</h2>
+      <ul>
+        <li>
+          {{ _('You can use Firefox and still have Chrome. It’s true, Chrome won’t change on your machine one bit.') }}
+        </li>
+
+        <li>
+          <details>
+            <summary>{{ _('Share with your friends how to switch to Firefox') }}</summary>
+
+            {% set share_url = settings.CANONICAL_URL + canonical_path %}
+
+            <ul class="share-options">
+              <li class="share-opt-facebook">
+                <a data-link-type="link" data-link-name="Share on Facebook" href="https://www.facebook.com/sharer/sharer.php?s=100&p[url]={{ share_url|urlencode }}">{{ _('Share to Facebook') }}</a>
+              </li>
+
+              <li class="share-opt-email">
+                <details>
+                  <summary>{{ _('Send an email') }}</summary>
+                  {# L10n: the string below is for a referral email subject line. #}
+                  {% set email_subject = _('Switch to Firefox')|urlencode %}
+
+                  {# L10n: the string below is for a referral email message body. Use \n\n for line breaks in the message as appropriate. #}
+                  {% set email_body = _('Hey, \n\nFirefox makes switching from Chrome really fast. I like it a lot, and you should try it. \n\nCheck it out and let me know what you think: ')|urlencode %}
```

Comment:
@flodolo also interested in your thoughts here on how best to handle this?

Reply:
Yes, this should work. What's after the colon though?

Reply:
After the colon comes the URL, which is populated automatically. Would an additional comment help here?

Reply:
Yep, it'd definitely help